.github/actions/contentful-teardown/action.yml

```diff
@@ -0,0 +1,40 @@
+name: Contentful Teardown
```

Comment:
Do we need this action? It's only being called in 1 place. We could inline it into the workflow.

Reply:
Potentially not, I put it here for readability as well as potential future reuse (potential use case where there WAS a migration on a PR but it gets removed -- seems edge-casey). Happy to move it.